### PR TITLE
utils_net: Add ovs bridge operations.

### PR DESCRIPTION
**Pull request by rbian**

### CODE REVIEW

File: virttest/utils_net.py [hunk 1]
```diff
@@ -1532,6 +1532,64 @@ def change_iface_bridge(ifname, new_bridge, ovs=None):
...
+    :param brname: Name of the bridge
+    :param ovs: OpenVSwitch object.
+    """
+    if ovs is None:
+        ovs = __ovs
+
+    if ovs is None:
+        raise OpenflowSwitchError(brname,
+                                  "There is no bridge in system.")
+    if brname in ovs.list_br():
```
Comment (will-Do) [inline virttest/utils_net.py]: Just a trick ```return brname in ovs.list_br()``` is enough

File: virttest/utils_net.py [hunk 2]
```diff
@@ -1532,6 +1532,64 @@ def change_iface_bridge(ifname, new_bridge, ovs=None):
...
+    :param brname: Name of the bridge
+    :param ovs: OpenVSwitch object.
+    """
+    if ovs is None:
+        ovs = __ovs
+
+    if ovs is None:
+        raise OpenflowSwitchError(brname,
+                                  "There is no bridge in system.")
+    if brname not in ovs.list_br():
```
Comment (will-Do) [inline virttest/utils_net.py]: So you can call the new fun you defined above: ```ovs_br_exists```

File: virttest/utils_net.py [hunk 3]
```diff
@@ -1532,6 +1532,64 @@ def change_iface_bridge(ifname, new_bridge, ovs=None):
...
+        ovs = __ovs
+
+    if ovs is None:
+        raise OpenflowSwitchError(brname,
+                                  "There is no bridge in system.")
+    if brname in ovs.list_br():
+        ovs.del_br(brname)
+    else:
+        raise BRNotExistError(brname, "")
+
```
Comment (will-Do) [inline virttest/utils_net.py]: And here

**Comment by rbian** [comment]
Thanks @will-Do,  updated the code.

File: virttest/utils_net.py [hunk 4]
```diff
@@ -1532,6 +1532,61 @@ def change_iface_bridge(ifname, new_bridge, ovs=None):
...
+    """
+    Check if bridge exists or not on OVS system
+
+    :param brname: Name of the bridge
+    :param ovs: OpenVSwitch object.
+    """
+    if ovs is None:
+        ovs = __ovs
+
+    if ovs is None:
```
Comment (chuanchang) [inline virttest/utils_net.py]: Need we judge if 'ovs' is None again?

Comment (rbian) [inline virttest/utils_net.py]: Yes, second one isn't needed. Thanks!

**Comment by will-Do** [comment]
ACK this PR

**Comment by will-Do** [comment]
BTW, pls rebase your patch

**Comment by rbian** [comment]
@will-Do , rebased.

**Comment by will-Do** [comment]
Oh, I mean squash your patches when rebase.

**Comment by waynesun09** [comment]
ACK and merging now.

**Pull request merged**